### PR TITLE
Added recipe for GOTTCHA

### PR DESCRIPTION
**Pull request by marcelmat12**
- [x] I have read the guidelines above.
- [x] This PR adds a new recipe.
- [ ] This PR updates an existing recipe.
- [ ] This PR does something else (explain below).

@bioconda/core

### CODE REVIEW

File: recipes/gottcha/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package:
+  name: gottcha
+  version: "1.0"
+
+source:
+  git_url: https://github.com/LANL-Bioinformatics/GOTTCHA.git
```
Comment (bgruening) [inline recipes/gottcha/meta.yaml]: can you pin please a release? This will take always the latest master. Ideally a tarball would be great.

**Comment by marcelmat12** [comment]
Hi Im somewhat stuck here
Could someone tell me what travis is doing in:
[https://travis-ci.org/bioconda/bioconda-recipes/jobs/171036404#L2469](url)

Because it gives out the message that the tool installed correctly :/

**Comment by daler** [comment]
It is saying it failed:
- https://travis-ci.org/bioconda/bioconda-recipes/jobs/171036402#L475
- https://travis-ci.org/bioconda/bioconda-recipes/jobs/171036402#L567). 

Looks like INSTALL.sh expects `unzip` to exist: https://travis-ci.org/bioconda/bioconda-recipes/jobs/171036402#L517

There's an `unzip` package in conda-forge you could include as a build requirement.

**Comment by marcelmat12** [comment]
Sorry to bother you again, Im sure you got more important stuff to do!

I added the `unzip` requirement and now on both Linux and OSX the building of tool is succesfull
As noted here:
[https://travis-ci.org/bioconda/bioconda-recipes/jobs/171322699#L2353](url)
[https://travis-ci.org/bioconda/bioconda-recipes/jobs/171322701#L2413](url)

I know that the travis fails, I just want to understand what the problem ist that causes travis to fail 

It always fails after this line: 
[https://travis-ci.org/bioconda/bioconda-recipes/jobs/171322699#L2419](url)
[https://travis-ci.org/bioconda/bioconda-recipes/jobs/171322701#L2478](url)

Thanks in advance!

**Comment by daler** [comment]
Now looking at the logs, it appears it's trying to [install a D compiler](https://travis-ci.org/bioconda/bioconda-recipes/jobs/171322699#L503) and some Perl packages (e.g. [YAML](https://travis-ci.org/bioconda/bioconda-recipes/jobs/171322699#L1753), but also Tie-IxHash and Module-Build). These should all be explicit dependencies in the GOTTCHA meta.yaml, and these may need their own recipes to be added if they don't already exist.

**Comment by marcelmat12** [comment]
Added the perl packaged as dependencies now! They are all on bioconda already.
Problem now is that there is no D-compiler as far as I saw
So looks like I will have to create recipe for it

**Comment by bgruening** [comment]
The unzip package is just python afaik. I tried to create one here: https://github.com/conda-forge/staged-recipes/pull/1719 help welcome!

**Comment by marcelmat12** [comment]
Hm ok but once i added the unzip package the Gottcha installation found it and was actually able to get further in the installation. The installation can now find all the dependencies it needs so it doesn't download any on its own except for the D-compiler

**Pull request closed without merging**